snap/local/runtime-helpers/bin/security-secretstore-setup-launch.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -ex
+
+security-secretstore-setup -confdir "$SNAP_DATA/config/security-secretstore-setup/res" --vaultInterval=10
+
+if [ -n "$REDIS5_PASSWORD_PATHNAME" ]; then
+
+    CONFIG_FILE="$SNAP_DATA/config/security-secretstore-read/res/configuration.toml"
+    # grep -q returns 0 if a match is found
+    grep -q "/tmp/edgex" "$CONFIG_FILE"
+    RETVAL="$?"
+
+    if [ $RETVAL == 0 ]; then
+        sed -e "s@/tmp/edgex/secrets/ca/ca.pem@$SNAP_DATA/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem@" \
+            -e "s@/tmp/edgex/secrets/edgex-redis/secrets-token.json@$SNAP_DATA/secrets/edgex-redis/secrets-token.json@" \
+            -e "s@/tmp/edgex/secrets/edgex-redis/redis5-password@$REDIS5_PASSWORD_PATHNAME@" \
+            -e "s@edgex-vault@localhost@" \
```

Comment:
why are we performing this sed expansion at runtime? I'd much rather do this at build-time or if really needed install-hook time, not at app service runtime